src/overlays/actors/ovl_En_Lift_Nuts/z_en_lift_nuts.h

```diff
@@ -7,11 +7,25 @@ struct EnLiftNuts;
 
 typedef void (*EnLiftNutsActionFunc)(struct EnLiftNuts*, PlayState*);
 
+#define ENLIFTNUTS_GET_FF00(thisx) (((thisx)->params & 0xFF00) >> 8)
+
 typedef struct EnLiftNuts {
     /* 0x0000 */ Actor actor;
```

Comment:
```suggestion
    /* 0x000 */ Actor actor;
```
Can you also delete the leading 0's from the rest as well?